BrowserKit/Sources/TabDataStore/TabDataStore.swift

```diff
@@ -58,7 +62,7 @@ public actor DefaultTabDataStore: TabDataStore {
             return nil
         }
         do {
-            let windowData = try await decodeWindowData(from: directoryURL)
+            let windowData = try fileManager.getWindowDataFromPath(path: directoryURL)
```

Comment:
not related to this PR specifically but I feel this func could use some documentation 🙏

Reply:
It has the documentation on the protocol. I can add more info if it's not clear? 
![image](https://user-images.githubusercontent.com/3929954/236470148-004e2fb8-9b27-42fb-9557-0f5df2a5019e.png)

Reply:
ohh sorry I meant documentation for `func fetchWindowData(withID id: UUID)` in `DefaultTabDataStore` I should've been more specific

Reply:
Ah ok, yeah I can add some documentation there. If it's ok I'll add it in my next PR so I can unblock BR from flaky tests sooner.